src/components/ChatPanel.tsx

```diff
@@ -30,8 +30,8 @@ const builtInCssClasses: ChatPanelCssClasses = withStylelessCssClasses(
   "Panel",
   {
     container: "h-full w-full flex flex-col relative shadow-2xl bg-white",
-    messagesScrollContainer: "flex flex-col mt-auto px-4 overflow-hidden",
-    messagesContainer: "flex flex-col gap-y-1 pb-[85px] overflow-auto",
+    messagesScrollContainer: "flex flex-col rounded-b-3xl mt-auto overflow-hidden",
```

Comment:
do we not need rounded for top as well? (e.g. in the case of no header component for example)

Reply:
Hm good point. The rounding is actually only required on the chat popup by default (to match the inputContainer rounding). So i've moved the styling to the chatpopup.tsx file

Reply:
also discussed this offline